paws_collector.js

```diff
@@ -263,6 +266,93 @@ class PawsCollector extends AlAwsCollector {
             return super.handleEvent(event);
         }
     };
+
+    // check the status of the state in DDB to avoid duplication
+    // there is a lot of logic here. not sur eif there is a better way of deduping. trying for an MVP
+    checkStateSqsMessage(stateSqsMsg, asyncCallback) {
+        const collector = this;
+        const DDB = new AWS.DynamoDB();
+        const tableName = process.env.pawsDDBTableName;
+
+        const params = {
+            Key: {
+                "CollectorId": {S: collector._collectorId},
```

Comment:
please add CID into the table because we'll be having collectors from multiple CIDs